mkdocs.yml

```diff
@@ -97,9 +102,10 @@ extra_javascript:
 
 plugins:
   - mkdocs-jupyter
-  #  - blog
+  - macros
```

Comment:
Why this addition? To support the Jinja2 templating mechanism?

Reply:
yes